config/rewrites.d/support.rackspace.com.json

```diff
@@ -114,5 +114,54 @@
         "rewrite": false,
         "status": 301
       }
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-accessing-cloud-files-containers/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-accessing-cloud-files-containers(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-browser-compatibility-matrix/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-browser-compatibility-matrix(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+{
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-add-and-delete-files-in-a-container/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-how-to-add-and-delete-files-in-a-container(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-remove-your-credentials-from-the-app/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-how-to-remove-your-credentials-from-the-app(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-set-up-credentials-to-access-cloud-files/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-how-to-set-up-credentials-to-access-cloud-files(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
+      {
+        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-overview/ URL to /how-to/cloud-files/",
+        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-overview(.*)$",
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
```

Comment:
Good morning, Ash. Thanks for the heads up on that.

-Kyle

From: Ash Wilson <notifications@github.com<mailto:notifications@github.com>>
Reply-To: rackerlabs/nexus-control <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, June 2, 2016 at 7:31 AM
To: rackerlabs/nexus-control <nexus-control@noreply.github.com<mailto:nexus-control@noreply.github.com>>
Cc: Rackspace Hosting <kyle.laffoon@RACKSPACE.COM<mailto:kyle.laffoon@RACKSPACE.COM>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [rackerlabs/nexus-control] Update support.rackspace.com.json (#479)


In config/rewrites.d/support.rackspace.com.json<https://github.com/rackerlabs/nexus-control/pull/479#discussion_r65530146>:

> +        "status": 301
> +      },
> +      {
> +        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person/ URL to /how-to/cloud-files/",
> +        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person(.*)$",
> +        "to": "/how-to/cloud-files/",
> +        "rewrite": false,
> +        "status": 301
> +      },
> +      {
> +        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-overview/ URL to /how-to/cloud-files/",
> +        "from": "^\\/how-to\\/article\\/cloud-storage-app-for-microsoft-sharepoint-overview(.*)$",
> +        "to": "/how-to/cloud-files/",
> +        "rewrite": false,
> +        "status": 301
> +      },


You'll also need to remove the final trailing comma here ☝️

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/rackerlabs/nexus-control/pull/479/files/0fcaedf468abaea5e2c27ac677a52cfbfd6e5c71#r65530146>, or mute the thread<https://github.com/notifications/unsubscribe/AFqHDB-_VzmkItK5LeADfmCauORP1weWks5qHs02gaJpZM4Ir_Fu>.